multiversx_sdk_cli/cli.py

```diff
@@ -45,9 +45,9 @@ def _do_main(cli_args: List[str]):
     args = parser.parse_args(argv_with_config_args)
 
     if args.verbose:
-        logging.basicConfig(level=logging.DEBUG)
+        logging.basicConfig(level=logging.DEBUG, force=True)
```

Comment:
Without `force = True`, already created logger objects (all of them, that is) won't have the new config.